lib/datatables_crud/datatables/active_record_datatable.rb

```diff
@@ -35,22 +35,31 @@ def count_options
 
       def count
         if params[:search].try(:[], :value).present? and search_columns.present?
-          @count ||= prepared_clazz.where(search_columns.map { |v| "#{v} like :search" }.join(' OR '), search: "%#{params[:search][:value]}%").count(count_options)
+          @count ||= prepared_clazz.where(search_columns.map { |v| "#{v} like :search" }.join(' OR '), search: "%#{params[:search][:value]}%").count
```

Comment:
`options` and `count_options` are there for static conditions - like you want to present a table of events with start date > 2019-11-19 etc. Then you'll see a table with records filtered according to these conditions and you will still have a possibility to use dynamic filtering with a matching string - `search`.